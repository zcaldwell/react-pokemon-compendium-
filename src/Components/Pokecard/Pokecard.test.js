import { render } from '@testing-library/react';
import PokeCard from './Pokecard';

test('renders the PokeCard', () => {
  const container = render(
    <PokeCard
      url_image="blah"
      pokemon="pikachu"
      type_1="electric"
      speed="slow"
      attack="blah blah"
      defense="blah"
    />
  );
  expect(container).toMatchSnapshot();
});
