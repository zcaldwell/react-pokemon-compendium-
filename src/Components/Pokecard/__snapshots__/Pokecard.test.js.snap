// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the PokeCard 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="pokecard"
      >
        <h2>
           Name: 
          pikachu
           
        </h2>
        <img
          src="blah"
        />
        <div
          class="stats"
        >
          <h3>
             Type: 
            electric
             
          </h3>
          <h3>
             Speed: 
            slow
             
          </h3>
          <h3>
             Attack: 
            blah blah
             
          </h3>
          <h3>
             Defense: 
            blah
             
          </h3>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="pokecard"
    >
      <h2>
         Name: 
        pikachu
         
      </h2>
      <img
        src="blah"
      />
      <div
        class="stats"
      >
        <h3>
           Type: 
          electric
           
        </h3>
        <h3>
           Speed: 
          slow
           
        </h3>
        <h3>
           Attack: 
          blah blah
           
        </h3>
        <h3>
           Defense: 
          blah
           
        </h3>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
